modules/agar/src/demo/ts/ephox/agar/demo/HtmlAssertionDemo.ts

```diff
@@ -3,18 +3,16 @@ import { Pipeline } from 'ephox/agar/api/Pipeline';
 import * as DemoContainer from 'ephox/agar/demo/DemoContainer';
 
 export const demo = (): void => {
-  DemoContainer.init(
-    'HTML Assertions',
-    (success, failure) => {
-
-      Pipeline.async({}, [
-        Assertions.sAssertHtml('Testing HTML', '<p>This sentence is slightly wrong</p>', '<p>This sentence is sightly wrng</p>')
-
-      ], success, (err) => {
+  DemoContainer.init('HTML Assertions', (success, failure) => {
+    Pipeline.async(
+      {},
+      [Assertions.sAssertHtml('Testing HTML', '<p>This sentence is slightly wrong</p>', '<p>This sentence is sightly wrng</p>')],
+      success,
+      (err) => {
```

Comment:
Definitely not a fan of how it re-arranges this, though that's mostly just because I'm used to how we normally do it. This looks very odd to me though.